File: Exercicio15.c
int main(int argc, char *argv[]){
  float peso,altura,imc;
  
  printf("Digite seu peso:");
  scanf("%f", &peso);
 
  printf("Digite sua altura: ");
  scanf("%f", &altura);

  imc = (peso/(altura*altura));


  if (imc < 20)
   {
      printf("Abaixo do peso");
  }

   else if (20<= imc && imc <25) {

       printf("Peso ideal");
  }

  else {
      
       printf("Acima do peso");
       }
     
     

  return 0;
}

File: Python/Exercicio15.py
altura = float(input('Digite a sua altura em (m): '))
 
peso = float(input('Digite o seu peso (em Kg): '))
 
imc = peso / (altura * altura)
 
if imc < 20:
    print("Abaixo do peso!")
elif imc >= 20 and imc < 25:
    print("Peso ideal!")
else:
    print("Acima do peso!")